proposals/2025/Add Loop.md

```diff
@@ -0,0 +1,336 @@
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="DesignReturnTemperature" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Design Return Temperature. (°C)</xs:documentation>
+                            </xs:annotation>
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="MinimumFlowFraction" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Minimum fraction of full flow allowed. (0-1)</xs:documentation>
+                            </xs:annotation>
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="FluidLoopOperationTypes" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Type of operation used by loop.</xs:documentation>
+                            </xs:annotation>
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:enumeration value="Continuous"/>
+                                    <xs:enumeration value="Intermittent"/>
+                                    <xs:enumeration value="Demand"/>
+                                    <xs:enumeration value="Scheduled"/>
+                                    <xs:enumeration value="Other"/>
+                                    <xs:enumeration value="Unknown"/>
+                                </xs:restriction>
+                            </xs:simpleType>
+                        </xs:element>
+                        <xs:element ref="auc:LinkedScheduleIDs" minOccurs="0"/>
+                        <xs:element name="FluidLoopFlowControlTypes" minOccurs="0">
```

Comment:
These should match the capitalization of preexisting enumerations (e.g., "Fixed Flow" and "Variable Flow").

Also, consider adding a "Two Position Flow" enumeration (c.f., https://github.com/BuildingSync/schema/blob/develop-v2/BuildingSync.xsd#L13908-L13921).